core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -345,6 +346,7 @@
     defineBinary(PLUS, Add, NullPolicy.STRICT, "plus");
     defineBinary(MINUS, Subtract, NullPolicy.STRICT, "minus");
     defineBinary(MULTIPLY, Multiply, NullPolicy.STRICT, "multiply");
+    defineBinary(MOD, Mod, NullPolicy.STRICT, "mod");
     defineBinary(DIVIDE, Divide, NullPolicy.STRICT, "divide");
```

Comment:
Oops, there is already a `MOD` implementation as the method call [1], so this patch is not necessary, sorry for that.
[1] https://github.com/apache/calcite/blob/2ac44153c4a521694c4dc975dd37b097184fc998/core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java#L353